### DISCUSSION TITLE
Edit motion.conf file on a Raspberry Pi

### DISCUSSION THREAD

**Discussion by bob59** [Motion-Project/motion#1814] [Q&A] [answered]
Raspberry Pi 3B+
Bullseye 32bit
motion 4.7.0
Hi, 
This is my first attempt at using motion, I have a working camera with motion detection and I can view the live stream on my network, but the recorded files are stored at /home/pi, I have been trying to save them in a folder I created 'Security.Camera'
I have tried edditing the line under System control configuration parameters
 ; target_dir value
 such as ; target_dir /home/pi/Security.Camera
but the files are still saved to /home/pi, could someone show me what I am doing wrong please
also before I try accessing the video stream remotely I wanted to set a user name and password, but I don't see an option to edit, do you have to add lines in the conf file to add features
Thanks
bob

**Comment by hormigaclub** [comment, marked as answer]
remove the semicolon

**Reply by bob59** [reply]
wow, that was simple, thank you
